### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by raieegulhane**
- [ ]  Added card component
Not responsive for phones or tabs.
image

### CODE REVIEW

**Review by sashaboi — APPROVED** [review]
Look good ! good to merge.

**Review by masraniJay — APPROVED** [review]
Looks good.

File: components/cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    padding: 1rem;
+    border-radius: var(--br-sq);
+}
+
+.card_container:hover {
+    box-shadow: 0 3px 20px 0 rgba(0, 0, 0, 0.5);
+}
+
+.card-vr {
+    display: flex;
```
Comment (shubhmehra) [inline components/cards/cards.css:14]: some places you have used dash & at some places you have used underscore.
please check & update that 
ex : card_container in above class & here it is card-vr

Comment (raieegulhane) [reply components/cards/cards.css:14]: Yes that is intentional. - is used for variant of the component. _ is used for section in the code for component

**Pull request merged**